opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java

```diff
@@ -100,7 +100,7 @@ protected void onSpanFinished(MockSpan mockSpan) {
     }
 
     @Override
-    public SpanBuilder buildSpan(String operationName) {
+    public Tracer.SpanBuilder buildSpan(String operationName) {
```

Comment:
hm, strictly speaking this _reduces_ the visibility since you're returning an interface even when you invoke the method directly via mock tracer instance. Was that your intention?